src/main/java/reposense/system/LogsManager.java

```diff
@@ -0,0 +1,109 @@
+package reposense.system;
+
+import java.io.IOException;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.FileHandler;
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.logging.SimpleFormatter;
+
+/**
+ * Configures and manages the loggers and handlers, including their levels
+ */
+public class LogsManager {
+    private static final int FILE_COUNT = 2;
+    private static final int MEGABYTE = (1 << 20);
+    private static final int MAX_FILE_SIZE_IN_BYTES = 5 * MEGABYTE; // 5MB
+    private static final String LOG_FILE = "reposense.log";
+
+    private static Level currentConsoleLogLevel = Level.INFO;
+    private static Level currentFileLogLevel = Level.INFO;
+    private static FileHandler fileHandler;
+    private static ConsoleHandler consoleHandler;
+
+    public static Logger getLogger(String name) {
+        Logger logger = Logger.getLogger(name);
```

Comment:
I think we should keep a list of all loggers created in `LogsManager`, so that later when we need to change the `currentConsoleLogLevel` based on user input, we can update each classes' logger.

Reply:
I think so, instead of maintaining a list, whenever a user initialises the logger, he can pass the level in the getLogger function!